homeassistant/components/twinkly/const.py

```diff
@@ -3,10 +3,10 @@
 DOMAIN = "twinkly"
 
 # Keys of the config entry
-CONF_ENTRY_ID = "id"
-CONF_ENTRY_HOST = "host"
-CONF_ENTRY_NAME = "name"
-CONF_ENTRY_MODEL = "model"
+ENTRY_DATA_ID = "id"
```

Comment:
I'd just do a `CONF_` prefix.
```suggestion
CONF_ID = "id"
```